src/teleop/jetson/jetson_teleop.py

```diff
@@ -0,0 +1,167 @@
+#!/usr/bin/env python3
+# Node for teleop-related callback functions
+
+from math import copysign
+import typing
+from enum import IntEnum
+import rospy as ros
+from sensor_msgs.msg import Joy, JointState
+from geometry_msgs.msg import Twist
+from mrover.msg import Chassis
+
+
+def quadratic(val):
+    return copysign(val**2, val)
+
+
+def deadzone(magnitude, threshold):
+    temp_mag = abs(magnitude)
+    if temp_mag <= threshold:
+        temp_mag = 0
+    else:
+        temp_mag = (temp_mag - threshold) / (1 - threshold)
+
+    return copysign(temp_mag, magnitude)
+
+
+def create_joint_msg(joints, joint, value):
+    joints[joint] = JointState()
+    joints[joint].velocity.append(value)
+    return joints
```

Comment:
I don't think you need to `return joints` here. Python is weird about passing variables but dictionaries should essentially be passed by reference iirc. Below, just let joints be updated automatically.

Reply:
Yea, just make sure to add a comment that the function modifies joints when you make the change